commerce_canadapost.module

```diff
@@ -128,3 +164,29 @@ function commerce_canadapost_commerce_shipment_form_submit($form, FormStateInter
   $form_state->setValues($values);
   $shipment->set('tracking_code', $submitted_tracking_pin);
 }
+
+/**
+ * Form submit for the commerce_store entity form.
+ */
+function commerce_canadapost_form_commerce_store_form_submit($form, FormStateInterface $form_state) {
+  $store = $form_state->getFormObject()->getEntity();
+
+  // Grab the form values.
+  $form_values = $form_state->getValues();
+
+  // If the user has opted to create Canada Post API settings for this store,
```

Comment:
personally, i feel like the code here is simple enough that the comments are unnecessary.

if we do keep them. maybe don't says "this store" 3 times in one sentence :P